contents/tutorials/next-steps-after-installing.md

```diff
@@ -0,0 +1,160 @@
+
+When working with others, you likely will want to [filter them out](/tutorials/filter-internal-users) of the events. To do this, go to ”project settings”, scroll down to “filter out internal and test users”  and add filters such as host does not include local development addresses and email does not include your company’s domain.
+
+## Data where you want it
+
+Once everyone’s onboard with PostHog, you want the data to be where you all can see it. You want events and dashboards top of mind to help you improve your product. To do this, you can setup webhooks and subscriptions to your data and dashboards. 
+
+If there are key events or actions you want to track, such as a potential customer visiting your pricing page, you can have it trigger a webhook. This requires setting up the webhook in the service of your choice (like [Slack](/docs/integrate/webhooks/slack), [Teams](/docs/integrate/webhooks/microsoft-teams), [Zapier](/tutorials/how-to-connect-posthog-and-notion-with-zapier), or [Discord](/docs/integrate/webhooks/discord)). You can then add the webhook to your PostHog instance in “project settings” under “webhook integration.” Finally, go to data management, create or select an action, then enable “post to webhook when this action is triggered.”
+
+![Webhook](../images/tutorials/next-steps-after-installing/hook.png)
+
+You can also subscribe to dashboards in Slack or via email. Just go to your dashboard, click the three dots to the left of share, click “subscribe,” and “new subscription.” If you want to send dashboards to Slack, you’ll need to configure our [Slack integration](/docs/integrate/third-party/slack). If you want dashboards by email, just fill in the details like the name, email, and recurrence, then click “create subscription.” 
+
+Finally, if you want your data backed up or sent to an external service like [BigQuery](/apps/bigquery-export), [S3](/apps/s3-export), [Snowflake](/apps/snowflake-export), or many more, you can use one of our many [“data-out” apps](/apps). Just go to “browse apps,” search for the one you want, configure it, and start sending the data to the other places you want it.
+
+![Data-out apps](../images/tutorials/next-steps-after-installing/apps.png)
+
+## Experiment to improve your product
+
+Once you have some data, and you’ve had a chance to gain some insights from it, it’s time to start using those insights to improve your product. A great way to do this is with experiments.
+
+Experiments is a feature that helps you test if changes improve your product in the ways you care about.
+
+To create an experiment, go to “experiments,” click “new experiment,” fill in the details, set up the variants you’d like to test, who’d you like to test it on, and the metric you’re trying to improve. When you click save, PostHog will create feature flags for your experiment that you can add to your code. Once you’ve implemented the feature flags, click launch and PostHog will track the experiment’s progress.
```

Comment:
This feels a bit thin as an explanation because it lacks detail about what info I should be adding, or what info is expected. It may be better to point to the docs or a separate tutorial.

Reply:
Rewrote to avoid details and added link to manual.